library/pod-security-policy/users/src.rego

```diff
@@ -22,31 +42,76 @@ accept_users("MustRunAsNonRoot", provided_user) = res {res := provided_user != 0
 
 accept_users("MustRunAs", provided_user) = res  {
   ranges := input.parameters.runAsUser.ranges
-  matching := {1 | provided_user >= ranges[j].min; provided_user <= ranges[j].max}
-  res := count(matching) > 0
+  res := is_in_range(provided_user, ranges)
```

Comment:
why not use `accept_value` here?

Reply:
`accept_value` uses `is_in_range` under the hood for `"MustRunAs"`. Using `is_in_range` directly reduces stack depth and means I don't need to echo `"MustRunAs"` into the calling of `accept_value` (as the way this function is defined means I don't have a variable for `field`, just the string)

Reply:
Interesting, sounds like this is a tradeoff between (minor) code duplication and (minor) stack depth increase. SGTM either way.